models/comments.models.js

```diff
@@ -1,6 +1,18 @@
 const db = require("../db/connection");
 const format = require("pg-format");
 
+exports.checkCommentIdExists = (commentId) => {
```

Comment:
We don't necessarily need a checkExists function for this one. We could instead check the response from the SQL query and ensure that it isn't an empty array